.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,23 @@
+name: build-pr
+
+on:
+  pull_request:
```

Comment:
on: [push, pull_request]

Reply:
decided to keep them separate so that the badge doesnt turn red if a PR build fails